src/types/types.ts

```diff
@@ -78,6 +78,22 @@ export type Crop = {
   cropMarker?: string
 }
 
+// `facetName` becomes mandatory when using `searchForFacetValues`
+export type SearchForFacetValuesParams = Omit<SearchParams, 'facetName'> & {
+  facetName: string
+}
+
+export type FacetHit = {
+  value: string
+  count: number
+}
+
+export type SearchForFacetValuesResponse = {
+  facetHits: FacetHit[]
+  facetQuery: string | null
+  processingTimeMs: number
```

Comment:
Is this only these 3 keys that will be responded?

Reply:
Yes! 
```
 {
      facetHits: [ { value: 'adventure', count: 1 } ],
      facetQuery: 'a',
      processingTimeMs: 0
    }
```